src/__a11y__/to-validate-a11y.ts

```diff
@@ -25,20 +25,40 @@ const htmlValidator = new HtmlValidate({
     'prefer-native-element': ['error', { exclude: ['listbox', 'button'] }],
     //TODO: revisit 'no-redundant-for' when fixing AWSUI-18968
     'no-redundant-for': 'off',
+    // innerHTML normalizes attribute values
+    // https://stackoverflow.com/questions/48092293/javascript-innerhtml-messing-with-html-attributes
+    'attribute-boolean-style': 'off',
   },
 });
 
 // Polyfill for Array.prototype.flatMap
-function flatMap<T>(arr: ReadonlyArray<T>, fn: (t: T) => any[]) {
-  return arr.reduce((acc: T[], item: T) => {
+function flatMap<Input, Output>(arr: ReadonlyArray<Input>, fn: (t: Input) => Output[]) {
```

Comment:
There were incorrect typings. Tests did not start locally unless I fixed this

Reply:
Do we still need this polyfill though? 

https://caniuse.com/?search=flatMap

Reply:
It all goes to this PR https://github.com/cloudscape-design/components/pull/144 (reverted in https://github.com/cloudscape-design/components/pull/169).

I am not going to change this in my PR